apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -457,11 +457,34 @@ export const DeeplinksProvider = ({
         },
         getStateFromPath: (path, config) => {
           const url = new URL(`ledgerlive://${path}`);
-          const { hostname, pathname } = url;
+          const { hostname, searchParams, href } = url;
+          let { pathname } = url;
+          const query = Object.fromEntries(searchParams);
+
+          const {
+            ajs_prop_campaign: ajsPropCampaign,
+            ajs_prop_track_data: ajsPropTrackData,
+            ajs_prop_source: ajsPropSource,
+            currency,
+            installApp,
+            appName,
+          } = query;
```

Comment:
Could there be any additional, not-yet-named `{... rest}` search params `query` data, which we might want to add to the `track` event?

Reply:
We want to avoid tracking all events by default, in order to prevent any leakage of sensitive information. This is the reason why we are creating ajsPropTrackData attribute.